lib/libutee/include/tee_api_defines.h

```diff
@@ -260,8 +260,170 @@
 #define TEE_ATTR_ECC_PRIVATE_VALUE          0xC0000341
 #define TEE_ATTR_ECC_CURVE                  0xF0000441
 
-#define TEE_ATTR_BIT_PROTECTED		    (1 << 28)
-#define TEE_ATTR_BIT_VALUE		    (1 << 29)
+#define TEE_ATTR_BIT_PROTECTED              (1 << 28)
+#define TEE_ATTR_BIT_VALUE                  (1 << 29)
+
+/* Panicked Functions Identification */
```

Comment:
These panic ids aren't used anywhere and there's no indication on how to use them. I would prefer if we wait with these defines until they are used (or at least some of them are used).